<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>setcoredump</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <widget class="QListView" name="listView">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>20</y>
         <width>421</width>
         <height>381</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="killbtn">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>420</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>kill</string>
       </property>
      </widget>
      <widget class="QPushButton" name="closebtn">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>420</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>close</string>
       </property>
      </widget>
      <widget class="QPushButton" name="refresh_all_btn">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>420</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>refresh all</string>
       </property>
      </widget>
      <widget class="QPushButton" name="refresh_xdesk_btn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>420</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>refresh xdesk</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
